static/gsApp/views/amCheckout/index.tsx

```diff
@@ -937,9 +936,15 @@ class AMCheckout extends Component<Props, State> {
                     <ExternalLink href="https://sentry.zendesk.com/hc/en-us/categories/17135853065755-Account-Billing" />
                   ),
                   contact: hasZendesk() ? (
-                    <ZendeskButton borderless onClick={activateZendesk}>
+                    <ZendeskStyledLink
+                      href="#"
+                      onClick={e => {
+                        e.preventDefault();
+                        activateZendesk();
+                      }}
+                    >
                       <Text variant="accent">{t('ask Support')}</Text>
-                    </ZendeskButton>
```

Comment:
i think we can keep the button and use the `link` [priority](https://sentry.sentry.io/stories/core/button#priorities) so we don't append a `#` to the browser url when this is clicked